### PR TITLE
Refine the documentation to include the 'hive_metastore' catalog and 'ucx' schema in all the output generated by the assignment tool

### PR DESCRIPTION
**Pull request by tamilselvanveeramani**

### CODE REVIEW

**Comment by tamilselvanveeramani** [comment]
Part of issue #199

File: src/databricks/labs/ucx/runtime.py [hunk 1]
```diff
@@ -28,18 +28,18 @@ def crawl_tables(_: MigrationConfig):
     storage location details.
 
     The gathered metadata is then subsequently organized and documented within a designated storage entity referred to
-    as the `$inventory.tables` table. This table serves as an extensive inventory, offering a well-structured and
-    readily accessible point of reference for users, data engineers, and administrators."""
+    as the `hive_metastore.ucx.tables` table. This table serves as an extensive inventory, offering a well-structured
```
Comment (nfx) [inline src/databricks/labs/ucx/runtime.py:31]: keep the `$inventory` variable here. See

https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/install.py#L280-L281

**Review by nfx — CHANGES_REQUESTED** [review]
why is this needed?

**Comment by tamilselvanveeramani** [comment]
as it's has been taken care by @nfx part of https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/install.py#L280-L281

**Pull request closed without merging**